src/TypeScript.UnitTests/NodeJSLocator/CompatibleNodeLocatorTests.cs

```diff
@@ -97,15 +130,18 @@ public void IsCompatibleVersion_ReturnsTrueFalse(int majorVersion, bool expected
             result.Should().Be(expectedResult);
         }
 
-        private CompatibleNodeLocator CreateTestSubject(IReadOnlyCollection<NodeVersionInfo> candidateLocations = null)
+        private CompatibleNodeLocator CreateTestSubject(IReadOnlyCollection<NodeVersionInfo> candidateLocations = null,
+            IUnsupportedNodeVersionNotificationService unsupportedNodeNotificationService = null)
         {
             candidateLocations ??= Array.Empty<NodeVersionInfo>();
             var versionInfoProvider = new Mock<INodeVersionInfoProvider>();
             versionInfoProvider.Setup(x => x.GetAllNodeVersions()).Returns(candidateLocations);
 
+            unsupportedNodeNotificationService ??= Mock.Of<IUnsupportedNodeVersionNotificationService>();
```

Comment:
Depends on `IUnsupportedNodeVersionNotificationService` being MEF-exportable - being done separately.

Reply:
Perhaps so this commit won't break, it's worth adding a dummy implementation of this service that is mef exportable.